CHANGES.md

```diff
@@ -4,9 +4,12 @@ Changelog
 Version 0.10.0
 -------------
 
-To be released.
```

Comment:
Space for release date has been removed. Seems better to let this line alone.

Reply:
I recovered it.